### PR TITLE
Bump proj version to remove OpenSSL dependency

### PR DESCRIPTION
**Pull request by urschrei**

### CODE REVIEW

**Comment by michaelkirk** [comment]
bors try

File: geo/CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changes
 
+## Unreleased
+
+* Bump proj version to 0.20.3
```
Comment (michaelkirk) [inline geo/CHANGES.md:5]: In the past, I haven't really paid attention to the release process, but shouldn't this should go in the 0.14.2 section which isn't yet released?

Comment (urschrei) [reply geo/CHANGES.md:5]: Oh, I didn't realise that we'd already bumped the version number in Cargo in preparation!

**Pull request merged**